server/homebrew.api.js

```diff
@@ -534,6 +553,7 @@ router.post('/api', checkClientVersion, asyncHandler(api.newBrew));
 router.put('/api/:id', checkClientVersion, asyncHandler(api.getBrew('edit', false)), asyncHandler(api.updateBrew));
 router.put('/api/update/:id', checkClientVersion, asyncHandler(api.getBrew('edit', false)), asyncHandler(api.updateBrew));
 router.delete('/api/:id', checkClientVersion, asyncHandler(api.deleteBrew));
+router.put('/api/prune/:id/:author', checkClientVersion, asyncHandler(api.deleteAuthor));
```

Comment:
should be able to put the `api.getBrew()` middleware inline here like we do on the other API calls. Then you don't need to do special handling inside `api.deleteAuthor`.

Reply:
Yes.